src/components/Exercise.stories.tsx

```diff
@@ -498,3 +519,23 @@ bitterness. The discriminant <span data-math="b^2 - 4ac"></span> could perhaps a
     </>
   );
 };
+
+export const Toolbar = () => {
```

Comment:
not sure I understand the difference between the Toolbar story and the Icons story